source/_docs/tools/quick-bar.md

```diff
@@ -0,0 +1,82 @@
+---
+title: "Quick Bar"
+description: "Dialog for quickly accessing entities or running common commands"
```

Comment:
I agree it reads better. I just have this fear that it will become an "omni-bar" that eventually holds every command under the sun. So I wanted to reinforce, early on, the idea that this should be a place for commonly-run commands.

But I think that's a "me problem" :) Your suggestion is good, let's do it.